pages/docs/global-configuration.md

```diff
@@ -26,7 +26,7 @@ function App () {
     <SWRConfig 
       value={{
         refreshInterval: 3000,
-        fetcher: (...args) => fetch(...args).then(res => res.json())
+        fetcher: (input, init) => fetch(input, init).then(res => res.json())
```

Comment:
```suggestion
        fetcher: (resource, init) => fetch(resource, init).then(res => res.json())
```

Let's change `input` to `resource` (or `endpoint`/`url`) since it's more descriptive, wdyt?

Reply:
Sure, I just followed the declaration names given by typescript in the `lib.dom.d.ts` here: https://github.com/microsoft/TypeScript/blob/master/lib/lib.dom.d.ts#L19866
```ts
declare function fetch(input: RequestInfo, init?: RequestInit): Promise<Response>;
``` 
What do you think?

Reply:
I see! I think it's still better to follow the MDN example though because if I was new to `fetch`, `resource` will be easier for me to understand that parameter is the key string.

Reply:
Understandable, mdn's Resource make more sense since the input data could be a `blob` or `data` etc..